tokio/src/time/clock.rs

```diff
@@ -0,0 +1,239 @@
+//! Source of time abstraction.
+//!
+//! By default, `std::time::Instant::now()` is used. However, when the
+//! `test-util` feature flag is enabled, the values returned for `now()` are
+//! configurable.
+
+#[cfg(feature = "test-util")]
+#[allow(unreachable_pub)] // https://github.com/rust-lang/rust/issues/57411
+pub use self::variant::{advance, pause, resume};
```

Comment:
Without the `test-util` flag, this passes through to `std::time::Instant`. With the flag, it adds additional capabilities.